packages/react-conformance/src/conformance.ts

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { Details } from './types';
+
+function isConformant(Component: React.ComponentType, options: Details = {}) {
+  const { constructorName } = options;
+
+  if (!constructorName) {
+    throw new Error('Constructor name is needed');
+  }
+
+  const infoJSONPath = `@fluentui/docs/src/componentInfo/${constructorName}.info.json`;
```

Comment:
We won't be using anything related to .info.json, so you should remove these parts.